src/ingestion/controllers/ingestionController.ts

```diff
@@ -19,8 +21,25 @@ export class IngestionController {
     private readonly ingestionManager: IngestionManager
   ) {}
 
-  public createLayer: RequestHandler = (req, res, next) => {
-    throw new Error('Method not implemented.');
+  public createLayer: NewLayerHandler = async (req, res, next) => {
+    try {
+      const newLayerRequestBody: unknown = req.body;
```

Comment:
why would you want the logger in the controller? it is not injected in the constructor and doesnt have real logic (the zod validator has logs of its own.)
if you want to print the request body before the zod validation i will add it but otherwise i will add the wanted logs in the manager

Reply:
another note: you dont have resolution in the metadata field - only in the part data.

Reply:
Ok, i agree - make sure access log in debug level and info level in manager file for each request (on logic start)

Reply:
access log should print the debug data so make sure it doesn't being printed twice